libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -206,6 +214,12 @@ public void onStop() {
     }
   }
 
+  @NonNull
```

Comment:
Could we add Javadoc for public API?

Reply:
No need here, it is just overrided method for LifecycleOwner